midi_app_controller/gui/binds_editor.py

```diff
@@ -462,6 +483,17 @@ def _wait_for_worker_threads(self):
         for thread in self.knobs_widget.thread_list:
             thread.wait()
 
+    def updateCheckBox(self):
```

Comment:
snake_case should be used for method names. And maybe something more verbose like `update_action_names_checkboxes`?